src/modules/core/components/MembersList/MembersListItem.tsx

```diff
@@ -67,9 +86,28 @@ const MembersListItem = <U extends AnyUser = AnyUser>(props: Props<U>) => {
         // eslint-disable-next-line jsx-a11y/no-noninteractive-tabindex
         tabIndex={onRowClick ? 0 : undefined}
       >
+        {userReputationData && tokenInfoData && (
```

Comment:
If you feel like it, you can wait until I merge #2298, rebase to pull in the changes, after which you will have access to actual reputation values.

With those values available to you, you can actually see how the reputation looks rendered as well as implement it as the original spec required you to, using a percent `%` value 

The above can be done by fetching the total reputation available in the colony, and calculating the current user's percentage out of it. Eg:
```
(<user reputation> / <total reputation>) * 100
```

Obviously, since those values will be displayed in `wei`, so you need to use `BigNumber` to calculate them.